Source/Scene/Particle.js

```diff
@@ -0,0 +1,82 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/Cartesian2',
+        '../Core/Cartesian3',
+        '../Core/Color'
+    ],function(
+        defaultValue,
+        defined,
+        Cartesian2,
+        Cartesian3,
+        Color) {
+    "use strict";
+
+    var Particle = function(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+
+        this.mass = defaultValue(options.mass, 1.0);
+        this.position = Cartesian3.clone(defaultValue(options.position, Cartesian3.ZERO));
+        this.velocity = Cartesian3.clone(defaultValue(options.velocity, Cartesian3.ZERO));
+        this.life = defaultValue(options.life, Number.MAX_VALUE);
+        this.image = defaultValue(options.image, null);
+        this.age = 0.0;
+        this.normalizedAge = 0.0;
+
+        this.startColor = defaultValue(options.startColor, Color.clone(Color.WHITE));
+        this.endColor = defaultValue(options.endColor, Color.clone(Color.WHITE));
+
+        this.startScale = defaultValue(options.startScale);
+        this.endScale = defaultValue(options.endScale);
+
+        var size = Cartesian2.clone(options.size);
+        if (!defined(size)) {
+            size = new Cartesian2(1.0, 1.0);
+        }
+
+        this.size = size;
+    };
+
+    Particle.prototype.update = function(forces, dt) {
+
+        // Apply the velocity
+        var delta = new Cartesian3();
+        Cartesian3.multiplyByScalar(this.velocity, dt, delta);
+        Cartesian3.add(this.position, delta, this.position);
+
+        // Update any forces.
+        var length = forces.length;
+        for (var i = 0; i < length; ++i) {
+            var force = forces[i];
+
+            if (typeof force === 'function') {
```

Comment:
One of the reasons I played with making Force an object in addition to a callback was so that forces could easily maintain their own state and be changeable.  I don't have a concrete example now but I thought about things like making gravity in the sandcastle example a class instead of a callback.

Something like:
```
var gravity = new LocalizedAcceleration(new Cartesian3(0.0, 0.0, -9.8));
forces = [gravity];
```

Then you could change it with:
```
gravity.acceleration = new Cartesian3(...);
```